src/snowflake/cli/api/utils/definition_rendering.py

```diff
@@ -232,6 +239,26 @@ def _validate_env_section(env_section: dict):
             )
 
 
+def _get_referenced_vars_in_definition(
+    template_env: TemplatedEnvironment, definition: Definition
+):
+    referenced_vars = set()
+
+    def find_any_template_vars(element):
+        referenced_vars.update(template_env.get_referenced_vars(element))
+
+    traverse(definition, visit_action=find_any_template_vars)
+
+    return referenced_vars
+
+
+def _template_version_warning(version):
+    cc.warning(
+        f"Possible use of templating in project definition file.\n"
+        f"Templating is not supported in current definition_version: {version}"
```

Comment:
In such messages, we should favour the resolution so that the user knows what the action should be. Something like "Ignoring template pattern in file <file>, update the project definition version to 1.1 or later in snowflake.yml to enable template expansion".

Reply:
Updated the message.